proxy/tests/transparency.rs

```diff
@@ -504,7 +504,6 @@ fn http1_bodyless_responses() {
     );
 
     assert_eq!(resp.status(), StatusCode::OK);
-    assert!(!resp.headers().contains_key("content-length"));
```

Comment:
These assertions aren't required, as it is allowed for these responses to include a content-length.